### ISSUE TITLE
New MOS supported?

### ISSUE THREAD

**Issue by kootstra** [henkwiedig/SiebelInstaller#1]
ran the scripts to create the CenOS image, provided my MOS details and the proceded with siebelinstaller.sh. However, I see no progress and wondered if with the new MOS this script is still expected to work. So I'm not going to spend time fixing the wrong things. 

Kind regards,

Anne

**Comment by henkwiedig** [comment]
Didn't test the script recently. However downloading from MOS might take some time. Look inside downloads folder for growing zip files. This would mean downloading is still in progress. 

Thee is no status/progress bar during the whole script run.
Later on steps like DB configure and Siebel seed data import run like forever inside a VM.
The wohle script runtime from a to z is +10 hours. 
Depends on hardware and internet speed.

**Comment by kootstra** [comment]
Thanks for the quick reply. Sadly the files in the Download directory aren't growing at all. The all stay at 0 bytes. When looking at the log files I can see the error in the log file that causes it: 

`https://updates.oracle.com/Orion/Services/download/p17470764_811_Linux-x86-64_2of26.zip?aru=16816067&patch_file=p17470764_811_Linux-x86-64_2of26.zip
Resolving updates.oracle.com... 141.146.44.51
Connecting to updates.oracle.com|141.146.44.51|:443... connected.
Unable to establish SSL connection.`

Which seems to be caused by the following line 36 of code in siebelinstaller_lib.sh:

`wget --no-check-certificate --user-agent="Mozilla/5.0" https://updates.oracle.com/Orion/Services/download`

This line works when I use wget64 on my windows machine, but not on CentOS images I downloaded: 6.4, 6.6 & 7.0. So I end up with a nice set of directories with files having 0 byte size.

Since this piece of code is based on the wget.sh file that can be downloaded from Oracle Support I tried to see if that would work, but sadly not. Even a standard wget.sh from Oracle Support  generates the same error. The code hasn't changed either so it should work OK.

**Comment by henkwiedig** [comment]
Here download and install worked fine over night.
Do you need a proxy for internet access ?

**Comment by kootstra** [comment]
Perhaps the only difference of note may be the host OS that I use. I run Virtualbox on Windows 8 though I have a hard time understanding how that would affect the functionality of `wget`.

Though experimentation I was able to make the download work. At present this process is underway and now the files are growing. The download site found in `create_vm.sh` is no longer valid. Perhaps your personal download directory still holds a copy, which it checks. Since it's an outdated version - and no longer supported - most mirrors no longer hold one. The [USC ftp server](http://mirrors.usc.edu/pub/linux/distributions/centos/6.4/isos/x86_64/) still holds a copy. 

The change I made to [siebelinstaller_lib.sh](https://github.com/henkwiedig/SiebelInstaller/blob/master/siebelinstaller_lib.sh) on line 29:

`WGET="/usr/bin/wget`**`--secure-protocol=TLSv1`** `--no-check-certificate"`

When the script has completed it's course I'll update here as well.
